hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/metrics/JRaftMetrics.java

```diff
@@ -65,10 +65,18 @@ public class JRaftMetrics {
     private JRaftMetrics() {
     }
 
-    public synchronized static void init(MeterRegistry meterRegistry) {
+    public synchronized static void init(MeterRegistry meterRegistry, boolean isInitialCall) {
         if (registry == null) {
             registry = meterRegistry;
             registerMeters();
+            if (isInitialCall) {
+                log.debug("JRaftMetrics initialized during application startup");
+            } else {
+                log.debug("JRaftMetrics initialized during scheduled refresh");
+            }
+        } else if (!isInitialCall) {
+            registerNodeMetrics();
```

Comment:
prefer to add a new method like resetNodeMetrics() instead of adding a isInitialCall param

Reply:
Thanks for your suggestion, I will optimize it